lib/sisimai/bite/email/gsuite.rb

```diff
@@ -0,0 +1,273 @@
+module Sisimai::Bite::Email
+  module GSuite
+    # Sisimai::Bite::Email::GSuite parses a bounce email which created by G Suite.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/GSuite.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/[@]googlemail[.]com[>]?\z/,
+        :subject => %r/Delivery[ ]Status[ ]Notification/,
+      }
+      Re1 = {
+        :begin   => %r/\A[*][*][ ].+[ ][*][*]\z/,
+        :error   => %r/\AThe[ ]response([ ]from[ ]the[ ]remote[ ]server)?[ ]was:\z/,
+        :html    => %r{\AContent-Type:[ ]*text/html;[ ]*charset=['"]?(?:UTF|utf)[-]8['"]?\z},
+        :rfc822  => %r{\AContent-Type:[ ]*(?:message/rfc822|text/rfc822-headers)\z},
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ErrorMayBe = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->